[2024-10-16T09:55:07.372+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-04-20T00:00:00+00:00 [queued]>
[2024-10-16T09:55:07.384+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-04-20T00:00:00+00:00 [queued]>
[2024-10-16T09:55:07.385+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:55:07.385+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T09:55:07.385+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T09:55:07.409+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-04-20 00:00:00+00:00
[2024-10-16T09:55:07.416+0000] {standard_task_runner.py:55} INFO - Started process 8532 to run task
[2024-10-16T09:55:07.424+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-04-20T00:00:00+00:00', '--job-id', '1667', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpj54hmd_a']
[2024-10-16T09:55:07.428+0000] {standard_task_runner.py:83} INFO - Job 1667: Subtask insert_rate
[2024-10-16T09:55:07.550+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-04-20T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T09:55:07.602+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T09:55:07.686+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-04-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-20T00:00:00+00:00
[2024-10-16T09:55:07.688+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 445.64950473, '2024-04-20')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T09:55:07.702+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:55:07.981+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T09:55:07.993+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 445.64950473, '2024-04-20')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T09:55:07.997+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T09:55:08.016+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240420T000000, start_date=20241016T095507, end_date=20241016T095508
[2024-10-16T09:55:08.069+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T09:55:08.089+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-16T10:05:11.748+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-04-20T00:00:00+00:00 [queued]>
[2024-10-16T10:05:11.794+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-04-20T00:00:00+00:00 [queued]>
[2024-10-16T10:05:11.796+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:05:11.801+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2024-10-16T10:05:11.802+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-16T10:05:11.856+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): insert_rate> on 2024-04-20 00:00:00+00:00
[2024-10-16T10:05:11.872+0000] {standard_task_runner.py:55} INFO - Started process 11118 to run task
[2024-10-16T10:05:11.888+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_usd_kzt_dag', 'insert_rate', 'scheduled__2024-04-20T00:00:00+00:00', '--job-id', '2403', '--raw', '--subdir', 'DAGS_FOLDER/exchange_rates/__init__.py', '--cfg-path', '/tmp/tmpj_5ou03k']
[2024-10-16T10:05:11.893+0000] {standard_task_runner.py:83} INFO - Job 2403: Subtask insert_rate
[2024-10-16T10:05:12.229+0000] {task_command.py:389} INFO - Running <TaskInstance: exchange_rate_usd_kzt_dag.insert_rate scheduled__2024-04-20T00:00:00+00:00 [running]> on host 63ee4fa64e74
[2024-10-16T10:05:12.351+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2024-10-16T10:05:12.691+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=exchange_rate_usd_kzt_dag
AIRFLOW_CTX_TASK_ID=insert_rate
AIRFLOW_CTX_EXECUTION_DATE=2024-04-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-20T00:00:00+00:00
[2024-10-16T10:05:12.697+0000] {sql.py:253} INFO - Executing: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 445.64950473, '2024-04-20')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;
[2024-10-16T10:05:12.724+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:05:13.608+0000] {base.py:73} INFO - Using connection ID 'postgres_default' for task execution.
[2024-10-16T10:05:13.634+0000] {sql.py:364} INFO - Running statement: INSERT INTO
    currency_exchange_rates
VALUES ('USD', 'KZT', 445.64950473, '2024-04-20')
ON CONFLICT (base, currency, date) DO
    UPDATE
        SET rate = excluded.rate;, parameters: None
[2024-10-16T10:05:13.644+0000] {sql.py:373} INFO - Rows affected: 1
[2024-10-16T10:05:13.701+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=exchange_rate_usd_kzt_dag, task_id=insert_rate, execution_date=20240420T000000, start_date=20241016T100511, end_date=20241016T100513
[2024-10-16T10:05:13.816+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2024-10-16T10:05:14.095+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
